Frontend/src/app/assets/asset.service.spec.ts

```diff
@@ -4,14 +4,17 @@
 /* tslint:disable:no-unused-variable */
 
 import { TestBed, inject } from '@angular/core/testing';
+// import { Observable } from 'rxjs';
```

Comment:
not needed?

Reply:
I fixed it thanks. "import { Subscription } from 'rxjs';" is used in angular 6+